libs/blocks/hero-marquee/hero-marquee.js

```diff
@@ -48,10 +52,31 @@ function decorateQr(el) {
   });
 }
 
-function decorateLockupFromContent(el) {
+async function loadIconography() {
+  await new Promise((resolve) => { loadStyle(`${base}/styles/iconography.css`, resolve); });
+}
+
+async function decorateLockupFromContent(el) {
   const rows = el.querySelectorAll(':scope > p');
   const firstRowImg = rows[0]?.querySelector('img');
-  if (firstRowImg) rows[0].classList.add('lockup-area');
+  if (!firstRowImg) return;
+  await loadIconography();
+  rows[0].classList.add('lockup-area');
+  rows[0].childNodes.forEach((node) => {
+    if (node.nodeType === 3 && node.nodeValue !== ' ') {
```

Comment:
Could the node value be `null`, case in which `node.nodeValue !== ' '` would be truthy?
```suggestion
    if (node.nodeType === 3 && node.nodeValue?.trim()) {
```

Reply:
Good call. I've made that changed and verified it works as expected.